src/styles/index.scss

```diff
@@ -1,95 +1,133 @@
         text-align: center;
         font-size: 40pt;
         font-weight: bold;
         margin-top: 25px;
     }
 
-    .timeline-container {
-        border: 1pt solid black;
-        margin: 5px 0px;
+    .timelines {
+        display: grid;
+        grid-template-columns: 55% 45%;
 
-        &:empty{
-            display: none;
-        }
-        
-        > .tweet-block {    
-            border: 1pt solid #c3c3c4;
-            padding: 20px 20px 10px;
+        .timeline-container {
+            margin: 0px $timeline-margin 0px;
             display: flex;
-            align-items: center;
-    
-            &:nth-of-type(odd){
-                background-color: #e8f5fd;
-            }
-        
-            &:nth-of-type(even){
-                background-color: #e9e9e9;
-            }
-        
-            &:last-child{
-                border-bottom-style: none;
+            flex-direction: column;
+            #timeline-button {
+                align-self: flex-end;
+                margin-bottom: 5px;
+                background-color: white;
+                border: 1px solid blue;
+                color: blue;
+                padding: 5px;
+            
+                &:hover{
+                    color: white;
+                    background-color: blue;
+                    cursor: pointer;
+                }
             }
 
-            .user-div  {
+            .header{
                 text-align: center;
-                margin-right: 30px;
-                flex: 0 0 15%; /*this sets user div's width to always be 15% regardless of flexing*/
-                > #user-icon {
-                    border-radius: 50%;
-                    height: 65px;
+            }
+
+            .tweets-container {
+                border: 1pt solid black;
+                margin: 5px 0px;
+                //flex: 0 0 40%;
+        
+                &:empty{
+                    display: none;
                 }
+
+                > .tweet-block {    
+                    border: 1pt solid #c3c3c4;
+                    padding: 20px 20px 10px;
+                    display: flex;
+
+                    &:nth-of-type(odd){
+                        background-color: #e8f5fd;
+                    }
                 
-                .user-name {
-                    font-weight: bold;
-                    font-size: 120%;
-                }
+                    &:nth-of-type(even){
+                        background-color: #e9e9e9;
+                    }
                 
-                .user-handle {
-                    font-size: 75%;
+                    &:last-child{
+                        border-bottom-style: none;
+                    }
+        
+                    .user-div  {
+                        text-align: center;
+                        margin-right: 30px;
+                        flex: 0 0 15%; /*this sets user div's width to always be 15% regardless of flexing*/
+                        > #user-icon {
```

Comment:
user-icon seems like something that would be reoccurring throughout your page/component. seems like you may have duplicate ids